### ISSUE TITLE
Even Kafka Connect is not enabled, still got "successfully create Kafka connect service"

### ISSUE THREAD

**Issue by jovezhong** [redpanda-data/console#1118]
I think the log message is a bit misleading
```
console-1  | {"level":"info","ts":"2024-02-15T01:09:39.561Z","msg":"started Redpanda Console","version":"v2.4.3","built_at":"1707418068"}
console-1  | {"level":"info","ts":"2024-02-15T01:09:39.561Z","msg":"creating Kafka connect HTTP clients and testing connectivity to all clusters"}
console-1  | {"level":"info","ts":"2024-02-15T01:09:39.561Z","msg":"tested Kafka connect cluster connectivity","successful_clusters":0,"failed_clusters":0}
console-1  | {"level":"info","ts":"2024-02-15T01:09:39.561Z","msg":"successfully create Kafka connect service"}
console-1  | {"level":"debug","ts":"2024-02-15T01:09:39.561Z","msg":"creating new kafka client","config":{..}
console-1  | {"level":"debug","ts":"2024-02-15T01:09:39.561Z","msg":"configuring SCRAM-SHA-256 mechanism"}
console-1  | {"level":"info","ts":"2024-02-15T01:09:39.561Z","msg":"connecting to Kafka seed brokers, trying to fetch cluster metadata"}
console-1  | {"level":"debug","ts":"2024-02-15T01:09:39.823Z","logger":"kafka_client_hooks","msg":"kafka connection succeeded","host":"..","dial_duration":0.077249916}
```

**Comment by bojand** [comment]
Fixed via #1122.

**Comment by jovezhong** [comment]
Thanks for the quick turnaround.
